asreview/config.py

```diff
@@ -63,10 +63,6 @@
     "notes": ["notes"],
     "keywords": ["keywords"],
     "doi": ["doi"],
-    "tags": [
-        "tags",
-    ],
-    "is_prior": ["asreview_prior", "is_prior"],
```

Comment:
We removed the Validation/Exploration project type as an explicit project type and integrated Validation with the Exploration/Review project type. 

Therefore, it is fine to deprecate the is_prior column here.